src/commands.ts

```diff
@@ -278,24 +278,23 @@ export const clone = async (scriptId: string, versionNumber?: number) => {
       },
     ]);
     scriptId = answers.scriptId;
-  } else {
-    // We have a scriptId or URL
-    // If we passed a URL, extract the scriptId from that. For example:
-    // https://script.google.com/a/DOMAIN/d/1Ng7bNZ1K95wNi2H7IUwZzM68FL6ffxQhyc_ByV42zpS6qAFX8pFsWu2I/edit
-    if (scriptId.length !== 57) {
-      // 57 is the magic number
-      const ids = scriptId.split('/').filter(s => {
-        return s.length === 57;
-      });
-      if (ids.length) {
-        scriptId = ids[0];
-      }
+  }
```

Comment:
Simply removed this else { } wrapping the logic. Not sure why the diff seems so large.

Reply:
By setting it you can hide whitespace changes.

<img width="769" alt="ss" src="https://user-images.githubusercontent.com/10114717/49592811-73ae4e00-f9b5-11e8-8d93-451b133ea721.png">